Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -450,7 +450,8 @@ void UpdateTextHtml()
 #if __MOBILE__
 			var attr = new NSAttributedStringDocumentAttributes
 			{
-				DocumentType = NSDocumentType.HTML
+				DocumentType = NSDocumentType.HTML,
+				StringEncoding = NSStringEncoding.UTF8
```

Comment:
I think a better solution would be to refactor this to a `protected virtual NSAttributedStringDocumentAttributes  GetNSAttributedStringDocumentAttributes` , that way developers can override that method and specify a different encoding if they need to.

Reply:
Sure :)